src/utils/ruleCalls.js

```diff
@@ -1,10 +1,19 @@
 import axios from "axios";
 
 export const listRules = async () => {
-  const response = await axios.get(process.env.API_URL);
+  const response = await axios.get(`${process.env.API_URL  }/bounce_rules`);
   if (response.status === 200) {
-    return await response;
-  } else {
-    throw new Error("Error retrieving all rules");
+    return response;
   }
+  throw new Error("Error retrieving all rules");
```

Comment:
would be nice to see the errors on the UI too - this can be a separate ticket. you'll need state for both errors and loading so you can show loading icons while loading and error messages on the UI if it fails to complete API calls.